logs_sdk/lib/opentelemetry/sdk/logs/logger.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # The SDK implementation of OpenTelemetry::Logs::Logger
+      class Logger < OpenTelemetry::Logs::Logger
+        attr_reader :instrumentation_scope, :logger_provider
```

Comment:
I'm curious why we expose these attributes. In the `Tracer` and `Meter`, they're private, so the inconsistency is a little weird.

Reply:
This `attr_reader` call has been removed. I'm not sure why I initially broke from `Tracer` and `Meter`.